src/components/board/LeadTitle.js

```diff
@@ -1,7 +1,11 @@
 import React from "react";
 
-const LeadTitle = () => {
-  return <div>LeadTitle</div>;
+const LeadTitle = (props) => {
+  return (
+    <div className="text-center text-2xl text-decoration-line: underline underline-offset-8">
```

Comment:
don't do underline since the line goes beyond the word. Do a horizontal line with a certain width. Check tailwind docs and check Figma to make it exactly like the design. 
Add font-montserrat so the font is correct as the Figma